kvbc/tools/db_editor/include/kv_blockchain_db_editor.hpp

```diff
@@ -392,6 +392,40 @@ struct GetCategoryEarliestStale {
   }
 };
 
+struct GetStaleKeysSummary {
+  const bool read_only = true;
+  std::string description() const {
+    return "getStaleKeysSummary [BLOCK-VERSION-TO]\n"
+           "  return the number of stale keys per category type in the current blockchain\n"
+           "If BLOCK-VERSION-TO has been given, the method will summerize the stale keys from current genesis to "
+           "BLOCK-VERSION-TO";
+  }
+
+  std::string execute(KeyValueBlockchain &adapter, const CommandArguments &args) const {
+    auto latestBlockID = adapter.getLastReachableBlockId();
+    if (args.values.size() >= 1) {
+      latestBlockID = toBlockId(args.values.front());
```

Comment:
I think we need to make sure that the argument's value is the range `[getGenesisBlockId(), getLastReachableBlockId()]`. I think we can add a test for it.